src/components/card/event-card/index.js

```diff
@@ -56,6 +53,10 @@ const EventCard = (props: Props) => {
   } = props;
 
   const [isBookmarked, setBookmark] = useState(bookmarked);
+  const productPropertiesList = [
```

Comment:
You can add id property to items of this list and then there is no reason to disable react/no-array-index-key in product/property-index

Reply:
I think it's useless to get an unnecessary parameter from component-user plus if the user doesn't know the use-case that we want this item, it may be entered repetitive id or don't enter it at all, but you're right it's not a good solution to disable linter for this file, I try to find another solution

Reply:
resolved in this commit > https://github.com/evandhq/react-jupiter/pull/25/commits/8474e0d09c26f88446e5426bc09f75fbe94a246b